src/Integrations/Util/Common.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace DDTrace\Util;
+
+use DDTrace\RootSpanData;
+use DDTrace\SpanData;
+
+class Common
+{
+    // When modifying this method, please also pay attention to the tests/ext/orphans.phpt test and update it accordingly if needed.
```

Comment:
Is there a reason why we cannot test that as part of an Integration test? I'd rather avoid duplicating logic like this.
It also doesn't really fit in tests/ext as it's not testing the extension either, but testing how this particular code here interacts with functionality the extension provides.

Reply:
I tried reaching out to the `/next-response` endpoint of the Request Replayer from TracerTestTrait, but it wouldn't work for w/e reason. I'm tempted to just use [the capabilities offered by the test agent](https://github.com/DataDog/dd-apm-test-agent/tree/master?tab=readme-ov-file#optional-agent_sample_rate_by_service) instead tbh

Reply:
Feel free to use that if it works :-)